samples/matter/temperature_sensor/src/default_zap/temperature_sensor.matter

```diff
@@ -2479,3 +2237,5 @@ endpoint 1 {
     ram      attribute clusterRevision default = 4;
   }
 }
+
+
```

Comment:
For now, you need to remove the additional line manually. I've created a PR that improves the west zap command to do it automatically.